Sources/Orbit/Support/Components/BarButton.swift

```diff
@@ -28,9 +29,16 @@ public struct BarButton: View {
         }
     }
 
-    public init(_ content: Icon.Content, size: Icon.Size = .large, alignment: TextAlignment = .center, action: @escaping () -> Void) {
+    public init(
+        _ content: Icon.Content,
+        size: Icon.Size = .large,
+        tint: Color? = nil,
```

Comment:
We can't get tint from environment, right?

Reply:
It is stupid, but sadly not.

https://kylehugh.es/documents/how-to-read-foreground-and-tint-colors-in-swiftui-views/